plugin/source/dynamix.unraid.net/usr/local/share/dynamix.unraid.net/scripts/api_utils.sh

```
@@ -43,20 +43,82 @@ get_archive_information() {
+  # Check if API config file exists
+  if [ ! -f "$api_config_path" ]; then
+    return 1
+  fi
+
+  # Check if jq is available
+  if ! command -v jq >/dev/null 2>&1; then
+    return 1
+  fi
+
+  # Parse JSON and check for plugin
+  local plugins=$(jq -r '.plugins[]?' "$api_config_path" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)